### PR TITLE
Annais

### PR DESCRIPTION
**Pull request by anarouanet**
don't accept

### CODE REVIEW

File: 02-how-to-contribute.Rmd [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# How to contribute
+# How to contribute -hello2
```
Comment (annahutch) [inline 02-how-to-contribute.Rmd]: hghgjhgj

Comment (anarouanet) [reply 02-how-to-contribute.Rmd]: I agree

Comment (annahutch) [reply 02-how-to-contribute.Rmd]: Sorry that was Kevin

**Comment by kkmann** [comment]
@annahutch @anarouanet if you don't want to merge thus PR, plz close.

**Pull request closed without merging**